spec/03-plugins/15-response-transformer/01-header_transformer_spec.lua

```diff
@@ -25,12 +25,11 @@ describe("Plugin: response-transformer", function()
       headers_sent = false,
       resp = {
       },
-      config = {
-        subsystem = "http",
-      },
+      config = ngx.config,
```

Comment:
If only the subsystem is present, `jit-uuid` will throw an error due to the absence of `nginx_configure`.

Reply:
Please add this comment in the code.